sdk/servicebus/azure-messaging-servicebus/src/samples/java/com/azure/messaging/servicebus/MessageSendAsyncSample.java

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.servicebus;
+
+import java.time.Duration;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+/**
+ * Sample demonstrates how to send {@link ServiceBusMessage} to an Azure Service Bus queue or topic.
+ */
+public class MessageSendAsyncSample {
```

Comment:
I thought you added this sample in another PR.

Reply:
Another PR I just had an update to it .

Reply:
This sample exists almost exactly: https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/servicebus/azure-messaging-servicebus/src/samples/java/com/azure/messaging/servicebus/SendMessageAsyncSample.java